ush/setup.py

```diff
@@ -1109,8 +1126,9 @@ def get_location(xcs, fmt, expt_cfg):
         for nco_var in nco_vars:
             nco_config[nco_var.upper()] = exptdir
 
-        # Set the rocoto string for the fcst output location
+        # Set the rocoto string for the fcst output location and comin directory
         rocoto_config["entities"]["FCST_DIR"] = "{{ nco.COMOUT_BASEDIR }}/@Y@m@d@H"
+        rocoto_config["entities"]["COMIN_DIR"] = "{{ nco.COMIN_BASEDIR }}/@Y@m@d@H"
```

Comment:
How about this one?

Reply:
@danielabdi-noaa, do you mean defining these from the workflow yaml file?

Reply:
Yes, these should also be easy enough to support from there. I think we should define workflow related stuff from the workflow yaml files alone.

Reply:
@chan-hoo I wonder why you needed to use format. Does it not render the jinja template correctly?

Reply:
@danielabdi-noaa, I just followed the existing one: 
'''
    LOGDIR: '{% if user.RUN_ENVIR == "nco" %}{{"{}/@Y@m@d".format(nco.LOGBASEDIR)}}{% else %}{{nco.LOGBASEDIR }}
'''
If you don't prefer it, I'll change it. However, I found another issue:
'''
       - !startstopfreq ['{%- if workflow.DATE_FIRST_CYCL != workflow.DATE_LAST_CYCL %}{{ [workflow.DATE_FIRST_CYCL[0:8], workflow.INCR_CYCL_FREQ]|join }}{%- else %}{{workflow.DATE_FIRST_CYCL}}{%- endif %}', '{{workflow.DATE_LAST_CYCL}}', '{{workflow.INCR_CYCL_FREQ}}']
'''
When I run the above def, the result of `[workflow.DATE_FIRST_CYCL[0:8], workflow.INCR_CYCL_FREQ]|join }}` becomes "202304286000" but it should "202304280600". How can I set workflow.INCR_CYCL_FREQ to 2-digits in the expression?

Reply:
@chan-hoo Formatting the cyle frequency probably works
```
"{:02d}".format(workflow.INCR_CYCL_FREQ)
```

Reply:
Btw, you don't have to do this in this PR :) It is just my wish list to have workflow stuff defined entirely from the workflow yaml files instead of using setup.py

Reply:
Thanks, and your suggestion works well!